### DISCUSSION TITLE
Gotify Update

### DISCUSSION THREAD

**Discussion by powellclan** [tteck/Proxmox#3255] [Help Needed]
I have a question re Gotify.  Given no update functionality has been provided as part of the installation script, I am wondering if anyone has discovered how to upgrade Gotify to the latest version.  Been trawling the web for a while with no luck so I thought I would try here for an answer.

Love these scripts btw, they have made my move to proxmox an absolute breeze.

**Comment by oliv99** [comment]
see my post https://github.com/tteck/Proxmox/discussions/3986

**Comment by havardthom** [comment]
Fixed in https://github.com/tteck/Proxmox/pull/3989, you can type `update` in LXC to update Gotify next time
